tests/test_kat.py

```diff
@@ -44,6 +44,8 @@ def test_sig_stfl(sig_stfl_name):
     if not(helpers.is_sig_stfl_enabled_by_name(sig_stfl_name)): pytest.skip('Not enabled')
     katfile = helpers.get_katfile("sig_stfl", sig_stfl_name)
     if not katfile: pytest.skip("KATs file is missing")
+    if not os.path.exists(katfile): # Skip if the file does not exist
+        pytest.skip(f'File not found: {katfile}')
```

Comment:
Is this good test practice? IMO a test should fail if a precondition fails (KAT file present in this case). This logic here seems to let the test pass instead (by skipping a test without KAT file). Skipping tests makes sense if an algorithm indeed is unavailable (3 lines above).